internal/cmd/integrate/integrate.go

```
@@ -104,6 +116,16 @@ func (c *integrateCmd) run() error {
 	return nil
 }
 
+// selectTools lets the user select the desired tools via an interactive multiselect dialog
+func selectTools() ([]string, error) {
+	selectedTools, err := dialog.MultiSelect("Select tools for integration", supportedTools)
+	if err != nil {
+		fmt.Printf("%+v \n", err)
```

The error shouldn't be printed to stdout. If we want to print it here, it should be to stderr - but since it's an unexpected error, IMO we should just return it with a stack trace.

---

It is the same way we handle the error in the `create` command (see `func (c *createCmd) getTestType() ` L125)
I would agree to change it... but then also in this location, too.

or maybe still returning it as an silent error and using `log.Debug`?